src/index.css

```diff
@@ -176,7 +176,8 @@ textarea {
  */
```

Comment:
please revert this file, then start again. Prettier was fixing this file and it keep changing it. now it has been ignored and it shouldn't get re generated if styles were not changed.

Reply:
@wisammechano  can you clarify this more I couldn't understand it well